src/main/java/com/bootcamp/demo/security/SecurityConfig.java

```diff
@@ -31,6 +31,8 @@ public void configure(final HttpSecurity http) throws Exception {
                 .and()
                 .logout()
                 .permitAll();
+
+        http.csrf().csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse());
```

Comment:
This is called above as well, so no need to duplicate it here as well. Probably was added when resetting the branch, but just as a note that it can be removed.